kernel/kernel-defaults/src/test/scala/io/delta/kernel/defaults/LogReplayEngineMetricsSuite.scala

```diff
@@ -445,6 +445,32 @@ class LogReplayEngineMetricsSuite extends AnyFunSuite with TestUtils {
     }
   }
 
+  test("checksum not found at the read version, but found at a previous version; " +
```

Comment:
Any reason we cannot just cherry-pick https://github.com/delta-io/delta/commit/d95d2a867ca8247075fbbf155d1a951554557941 ?

Reply:
Makes sense then, thanks!